src/pages/About/AboutPage.jsx

```diff
@@ -1,5 +1,170 @@
+            <S.SubTitle>Members</S.SubTitle>
+            <S.SubTitleKr>학회원</S.SubTitleKr>
+          </S.SubTitleContentContainer>
+          <S.AboutContainer>
+            <S.CardContainer>
+              <CardContent name='YB' subTitle='신입 학회원' />
+              <CardContent name='OB' subTitle='한 학기 수료 학회원' />
+              <CardContent name='Alumni' subTitle='수료 학회원' />
+            </S.CardContainer>
+            <S.TextWrapper>
+              {`EPRIS의 활동 멤버는 OB와 YB로 구성됩니다. 모든 학회원은 2학기 연속 활동을 원칙으로 합니다.
+
+EPRIS는 기획부, 운영부, 홍보부로 구성되어 운영됩니다. 각 부서는 부장과 부원들로 구성되며, 활동 멤버 전원은 하나의 부서에 소속되어 활동합니다.
+
+EPRIS의 임원진은 회장, 기획부장, 운영부장, 홍보부장의 4인 체제로 구성됩니다. 임원진은 한 학기 이상 활동을 수료한 회원으로 구성됩니다.`}
+            </S.TextWrapper>
+          </S.AboutContainer>
+        </S.SubTitleContainer>
+
+        <S.SubTitleContainer>
+          <S.SubTitleContentContainer>
+            <S.SubTitle>Administrating Team</S.SubTitle>
+            <S.SubTitleKr>부서 소개</S.SubTitleKr>
+          </S.SubTitleContentContainer>
+          <S.DepartmentContainer>
+            <S.HiddenCardContainer>
+              <S.CardContentContainer>
+                <S.CardName>기획부장</S.CardName>
+                <S.SubTitleKr>기획부 총괄</S.SubTitleKr>
+              </S.CardContentContainer>
+              <S.CardContentContainer>
+                <S.CardName>운영부장</S.CardName>
+                <S.SubTitleKr>운영부 총괄</S.SubTitleKr>
+              </S.CardContentContainer>
+              <S.CardContentContainer>
+                <S.CardName>홍보부장</S.CardName>
+                <S.SubTitleKr>홍보부 총괄</S.SubTitleKr>
+              </S.CardContentContainer>
+              <S.CardContentContainer>
+                <S.CardName>회장</S.CardName>
+                <S.SubTitleKr>학회 총괄</S.SubTitleKr>
+              </S.CardContentContainer>
+            </S.HiddenCardContainer>
+            <M.CardContainer>
+              <M.CardButtonContainer>
+                <M.LeftArrow onClick={handlePrevClick} />
+                <M.CardContentContainer>
+                  <S.CardName>{cardData[currentCardIndex].title}</S.CardName>
+                  <S.SubTitleKr>
+                    {cardData[currentCardIndex].subtitle}
+                  </S.SubTitleKr>
+                </M.CardContentContainer>
+                <M.RightArrow onClick={handelNextClick} />
+              </M.CardButtonContainer>
+            </M.CardContainer>
+            <S.Conatiner>
+              {departmentData.map((department, index) => (
+                <React.Fragment key={index}>
+                  <div style={{ height: '19.8125rem' }}>
+                    <S.Triangle marginLeft={department.marginLeft} />
+                    <DepartmentCardContent
+                      marginLeft={department.marginLeft}
+                      depname={department.depname}
+                      description={department.description}
+                      depdetails={department.depdetails}
+                    />{' '}
+                  </div>
```

Comment:
```suggestion
                  <S.Triangle marginLeft={department.marginLeft} />
                  <DepartmentCardContent
                    marginLeft={department.marginLeft}
                    depname={department.depname}
                    description={department.description}
                    depdetails={department.depdetails}
                  />
```
모바일일때 높이 고정이라 style 파일에 M.Container에 높이 추가해 놓았습니다